tests/tests.go

```diff
@@ -247,7 +281,40 @@ func (ctl *controller) addInstance(instanceCreate api.InstanceCreate) (*api.Inst
 
 	instance := new(api.Instance)
 	if err = jsonapi.UnmarshalPayload(body, instance); err != nil {
-		return nil, fmt.Errorf("fail to unmarshal post instance response, error %w", err)
+		return nil, fmt.Errorf("fail to unmarshal post instance response, error: %w", err)
 	}
 	return instance, nil
 }
+
+// createIssue creates an issue.
+func (ctl *controller) createIssue(issueCreate api.IssueCreate) (*api.Issue, error) {
+	buf := new(bytes.Buffer)
+	if err := jsonapi.MarshalPayload(buf, &issueCreate); err != nil {
+		return nil, fmt.Errorf("failed to marshal issue create, error: %w", err)
+	}
+
+	body, err := ctl.post("/issue", buf)
+	if err != nil {
+		return nil, err
+	}
+
+	issue := new(api.Issue)
+	if err = jsonapi.UnmarshalPayload(body, issue); err != nil {
+		return nil, fmt.Errorf("fail to unmarshal post issue response, error: %w", err)
+	}
+	return issue, nil
+}
+
+// getDatabaseCreateIssueCreateContext gets a create context for create database issue.
+func getDatabaseCreateIssueCreateContext(instanceID int, databaseName string) (string, error) {
```

Comment:
nit: return CreateDatabaseContext and let caller decides?

Reply:
Inlined with caller then since the logic isn't complex. Was trying to save some json marshalling boiler code.